wiki/Community/Project_Loved/en.md

```diff
@@ -31,15 +31,13 @@ There are some very minimal criteria that need to be met for beatmaps to be nomi
 - At least one difficulty can be passed
 - Every nominated difficulty has a [drain time](/wiki/Beatmap/Drain_time) of at least 20 seconds
 
-Although many unranked beatmaps meet these criteria, few are voted into the Loved category due to the selective nature of the captains' picks, and the high community approval requirement by voting.
```

Comment:
why remove this? It's still accurate to the point it feels like a warranted disclaimer.

Reply:
Thought it was unnecessary since it's obvious that not ever map can have it's turn, but if you rly want to I will add it back in there.

Reply:
ok